core/modules/date/date.admin.inc

```diff
@@ -197,7 +197,7 @@ function _date_field_instance_settings_form($field, $instance) {
   $form['default_value'] = array(
     '#type' => 'select',
     '#title' => t('Default date'),
-    '#default_value' => $settings['default_value'],
+    '#default_value' => 'blank',//$settings['default_value'],
```

Comment:
I think this will cause Date to revert to no default value every time you revisit the settings form. Rather than here in the form, the default should be changed in `date_field_info()`.